src/DocumentFormat.OpenXml/SimpleTypes/OpenXmlComparableValueType.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace DocumentFormat.OpenXml
+{
+    /// <summary>
+    /// Represents a comparable and equatable value for simple value types (Int32, UInt32, Byte, struct, etc).
+    /// </summary>
+    /// <typeparam name="T">The type of the value.</typeparam>
+    public abstract class OpenXmlComparableValueType<T> : OpenXmlSimpleValue<T>,
```

Comment:
As with the previous comment for the reference type, I think `OpenXmlComparableSimpleValue` would fit better with the current naming

Reply:
If we were to derive such as `OpenXmlComparableSimpleValue<T> : OpenXmlComparableSimpleType<T?>` would that work?